examples/weather/unified_recipe/train.py

```diff
@@ -432,6 +433,20 @@ def train_step_forward(
 
         # Finish training
         if dist.rank == 0:
+            # Save model card
+            logger.info("Saving model card")
+            save_inference_model_package(
+                model,
+                cfg,
+                predicted_variable_normalizer=predicted_batch_norm,
+                unpredicted_variable_normalizer=unpredicted_batch_norm,
+                latitude=zarr.open(cfg.dataset.dataset_filename, mode="r")["latitude"],
```

Comment:
Is `zarr.open` fast? If not, maybe call it once and store the result?
```py
with zarr.open(cfg.dataset.dataset_filename, mode="r") as zarr_store:
...
    latitude=zarr_store["latitude"]
```

Reply:
Its actually extremely fast so I got a bit lazy doing it when I might not need to.